third_party/3/packaging/specifiers.pyi

```diff
@@ -0,0 +1,70 @@
+import abc
+from typing import Callable, FrozenSet, Iterable, Iterator, Optional, Union
+
+from .utils import canonicalize_version as canonicalize_version
+from .version import LegacyVersion as LegacyVersion, Version as Version, parse as parse
+
+ParsedVersion = Union[Version, LegacyVersion]
```

Comment:
These should be marked as private because they don't exist at runtime (they're defined in an `if TYPE_CHECKING` block).